libpod/container_internal_linux.go

```diff
@@ -266,9 +266,15 @@ func (c *Container) setupSystemd(mounts []spec.Mount, g generate.Generator) erro
 		g.AddMount(systemdMnt)
 	} else {
 		mountOptions := []string{"bind", "rprivate"}
+		skipMount := false
 
 		var statfs unix.Statfs_t
 		if err := unix.Statfs("/sys/fs/cgroup/systemd", &statfs); err != nil {
+			if os.IsNotExist(err) {
```

Comment:
please use errors.Is

Reply:
I always forget this...

Reply:
Too bad. I wonder if there is linter for this?

Reply:
comment addressed and pushed a new version